src/Compiler/Syntax/Tokenizer.Toklets.cs

```diff
@@ -415,11 +415,22 @@ private Token ConsumeEOFToken(in char c)
                         tokenList.Add(currToken);
                 }
 
-                this.Reconsume();
+                if (tokenList.Count > 0) {
+                    sections.Add(tokenList.ToImmutable());
+
+                    output.Append('{').Append(sections.Count - 1).Append('}');
+                } else {
+                    Logger.Error(new UnexpectedError<char>(ErrorArea.Tokenizer) {
+                        In = "an interpolated string",
+                        Value = '}',
+                        Expected = "an expression",
```

Comment:
```suggestion
                        In = "an interpolated string",
                        Value = '}',
                        Expected = "a value",
                        Message = "An interpolated section can't be empty.",
```
I feel like adding a little message saying it can't be empty would be a good idea, even though it's kinda obvious what the problem is.